kuma/static/styles/components/compat-tables/bc-icons.scss

```diff
@@ -4,17 +4,26 @@
 .bc-browsers {
     .bc-head-txt-label {
         display: inline-block;
-        margin: 0 0 0 50px;
+        bottom: 0;
+        margin: 0 0 20px 10px;
         padding: 0;
-        width: 140px;
-        height: 210px;
+        width: 20px;
+        height: 20px;
         transform: rotate(-90deg);
+        transform-origin: left;
```

Comment:
Hmm. I'm used to rotating around the center or around a corner. Maybe this rotate around the midpoint of an edge has something to do with the asymmetrical margins...  I can't visualize what is going on here, but I can see from your screenshots that it worked.

Reply:
Yup, on this point and the previous. At this point rotating around the left origin resolves the positioning problem. I would like to revisit the BCD table CSS at some point and improve the overall CSS so there are fewer surprises like these.